Makefile

```diff
@@ -317,6 +320,32 @@ bundle-base: manifests kustomize operator-sdk ## Generate bundle manifests and m
 
 export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
 
+.PHONY: add-console-plugin-annotation
+add-console-plugin-annotation: ## Add console-plugin annotation to the CSV
+	# add console-plugin annotation
```

Comment:
I think this line is redundant when adding the annotation is the only thing that is done under this target